recipes/tmap/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "tmap" %}
+{% set version = "1.0.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/reymond-group/tmap/archive/9aa043dd1af022161048ee0a4b5554ec5e34977f.zip
+  sha256: 56cf4f83d663950ab7ef41deb425b8e1c01de378fb5239aeef7b6613a6edf212
+
+  patches:
+    - 01_bind2d_deprecated.diff
+    - 02_setup_windows.diff
+
+build:
+  number: 0
+  skip: True  # [win]
+  binary_relocation: False  # [osx]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - python {{ python }}
+    - cmake >=3.1
+    - make  # [not win]
+    - ninja  # [win]
+    - llvm-openmp  # [osx]
+    - libgomp  # [linux]
+    - openmp  # [win]
+    - setuptools
+    - pybind11
+    - ogdf ==2020.02
+  host:
+    - vc  # [win]
+    - llvm-openmp  # [osx]
+    - libgomp  # [linux]
+    - openmp  # [win]
+    - python
+    - pip
+  run:
+    - vc  # [win]
+    - llvm-openmp  # [osx]
+    - libgomp  # [linux]
+    - openmp  # [win]
+    - python
+    - ogdf
+
+test:
+  imports:
+    - tmap
+
+about:
+  home: https://github.com/reymond-group/tmap
+  license: MIT
```

Comment:
(BTW: Whenever you distribute `tmap`, it will have to be GPL licensed due to OGDF's license.)

Reply:
Only on link though

Reply:
And this should be dynamically linked, so ogdf is not redistributed as part of tmap here

Reply:
Please confirm me what to do here.

Reply:
I believe this is fine as is.  This is what we do for every other package on conda-forge.  It is up to users and downstream redistributors of the code/environments to be aware of the license restrictions and how redistribution of bundles need to be licensed.

Reply:
ooo but you are building static libs.  Sorry @mbargull.  You should be building dynamic libs to avoid this

Reply:
So, if it's statically linked, then `tmap` has to be GPL (v2 or v3) licensed for all "interpretations of the GPL".
If it's dynamically linked, then `tmap` has to be GPL (v2 or v3) licensed, at least acc. to the FSF (see https://www.gnu.org/licenses/gpl-faq.html#GPLStaticVsDynamic).

I am not supportive of the FSF's intentions. However, I'd say licensing `tmap` under the GPL is the safest option then.

----
If you really would like to avoid licensing `tmap` under the GPL, you could ask the OGDF maintainers for a linking exception. Unless things have changed over the years, they are rather open to granting those exceptions.

Reply:
With dynamic linking, the linking happens on the users machine. The actual objects here are not linked and the license should remain what the authors published under. The combined work only exists on the users machine, not in the `.tar.bz2` file

Reply:
Keeping MIT here since the lib is shared now. Reopen if needed.